ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/translator/ASTBuilder.java

```diff
@@ -58,14 +73,23 @@ public static ASTNode destNode() {
                 "TOK_TMP_FILE")).node();
   }
 
-  public static ASTNode table(RelNode scan) {
-    HiveTableScan hts;
-    if (scan instanceof DruidQuery) {
-      hts = (HiveTableScan) ((DruidQuery)scan).getTableScan();
+  public static ASTNode table(final RelNode scan) {
+    JdbcHiveTableScan jdbcHiveTableScan = null;
```

Comment:
This variable can be removed as it is not used later on. Just cast within the ```if``` clause.